### PR TITLE
Add the SellBuilding Unique

### PR DESCRIPTION
**Pull request by PLynx01**
From this PR #10431 

I would like to add the variation of removeBuilding unique (added by me). The difference is that SellBuilding gives Gold for removed buildings.

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/unique/UniqueType.kt [hunk 1]
```diff
@@ -131,6 +131,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     GainFreeBuildings("Gain a free [buildingName] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
     BuildingMaintenance("[relativeAmount]% maintenance cost for buildings [cityFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
     RemoveBuilding("Remove [buildingFilter] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
+    SellBuilding("Sell [buildingFilter] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unique/UniqueType.kt:134]: Looking at this again, these should be phrased "Sell [buildingFilter] buildings [cityFilter]" - that way we can say e.g. "Sell [Culture] buildings [in all cities]" and it looks correct

Comment (PLynx01) [reply core/src/com/unciv/models/ruleset/unique/UniqueType.kt]: Done

File: docs/Modders/uniques.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ Simple unique parameters are explained by mouseover. Complex parameters are expl
 
 	Applicable to: Triggerable, Global
 
+??? example  "Sell [buildingFilter] [cityFilter]"
+	Example: "Sell [Culture] [in all cities]"
```
Comment (karmaBonfire) [inline docs/Modders/uniques.md:22]: Looks like docs weren't updated with your latest changes:
```suggestion
??? example  "Sell [buildingFilter] buildings [cityFilter]"
	Example: "Sell [Culture] buildings [in all cities]"
```

Comment (yairm210) [reply docs/Modders/uniques.md:22]: That's fine, version release will update those docs automagically :)

**Review by yairm210 — APPROVED** [review]

**Pull request merged**